Sources/SmileID/Classes/Helpers/LocalStorage.swift

```diff
@@ -20,12 +22,30 @@ public class LocalStorage {
         }
     }
 
+    static var pendingDirectory: URL {
```

Comment:
Any reason this can't just be a constant instead of a getter? i.e. something like `static let pendingDirectory = defaultDirectory.appendingPathComponent`

Reply:
It can `throw`, and we can't assign to a constant if it throws